README.md

```diff
@@ -110,41 +110,74 @@ This sample shows how to create custom AnalyzeRules for use within the Analyze w
-   - Assign all content intended for "install-time" delivery to the "InstallTimeContent" asset pack. This is a "placeholder" asset pack that is representative of the streaming assets pack. No custom asset pack named "InstallTimeContent" is actually created.
-6. In the “Content Packing & Loading” schema:
-   1. Set the “Build & Load Paths” to the default local paths (LocalBuildPath and LocalLoadPath). At runtime we will configure the load paths to use asset pack locations (see AddressablesInitSingleton.cs).
-      - Since the Google Play Console doesn't provide remote URLs for uploaded content, it is not possible to use remote paths or the Content Update workflow for content assigned to asset packs. Remote content will need to be hosted on a different CDN.
-   2. In Advanced Options > Asset Bundle Provider use the “Play Asset Delivery Provider”. This will make sure that asset packs are downloaded before loading content from AssetBundles.
-7. Build Addressables using the custom “Play Asset Delivery” build script. In the Addressables Groups Window, do Build > New Build > Play Asset Delivery.
-   1. Prepare each bundle file to be assigned to its own asset pack:
-      - Each asset pack will have a directory named “{asset pack name}.androidpack” in “Assets/PlayAssetDelivery/CustomAssetPackContent".
-      - Each .androidpack directory also contains a ‘build.gradle’ file that specifies that delivery type for the asset pack. If this file is missing, Unity will assume that the asset pack’s delivery type is “on-demand”.
-   2. Create a "CustomAssetPacksData.json" file in 'Assets/StreamingAssets' that stores all custom asset pack information to be used at runtime.
-8. Prepare Addressables to load content from asset packs at runtime (we do this in AddressablesInitSingleton.cs).
-   1. Make sure that the generated asset packs are downloaded.
-   2. Configure our custom InternalIdTransformFunc, which converts internal ids to their respective asset pack location.
-   3. Load all custom asset pack data from the "CustomAssetPacksData.json".
-   4. Then load assets using the Addressables API (we do this in LoadObject.cs).
-9. When ready to build the Android App Bundle, open File > Build Settings and click “Build”. This will create all of our custom asset packs along with the generated asset packs.
-   - If you want to upload the App Bundle to the Google Play Console, make sure that you are doing a release build.
+
+##### Configure custom Addressable scripts
+The sample project uses custom Addressable scripts that make it easier to build and load AssetBundles assigned to asset packs.
+- In 'Assets/PlayAssetDelivery/Editor' we have:
```

Comment:
Replace with:
"The 'Assets/PlayAssetDelivery/Editor' directory contains the following scripts:"